examples/braket_features/Native_gates_on_IonQ.ipynb

```diff
@@ -0,0 +1,1666 @@
+{
```

Comment:
<p>Kind of weird that we're showcasing this printing of a circuit since it's so hard to read anyway.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>

Reply:
yeah I wasnt sure about this. I'll cut the print statement.